app/views/jobs/show.html.slim

```diff
@@ -1,16 +1,24 @@
-#job.job-posting
-  .job-posting__header
-    h2.job-company=@job.company
-    h1.job-title=@job.title
-    span.job-posting-date=@job.posted_at
-    span.job-availability=@job.availability
-    .job-share=mail_to "", "Share this job", 
-      {subject: "[Chadev+Jobs] #{@job.title}", 
-        body: "Here is a job I thought you might be interested in. <br/><br/> #{job_url(@job)}"}
+- if @job.akismet_spam
```

Comment:
I renamed the attribute, but after I was done fixing up all references to the old attribute name, it ended up messier than I had hoped. I think I'll stick with the non-predicate-method `akismet_spam` attribute.